system_tests/speech.py

```diff
@@ -125,6 +125,32 @@ def test_sync_recognize_gcs_file(self):
                                          max_alternatives=1)
         self._check_best_results(result)
 
+    def test_sync_recognize_local_file_rest(self):
+        with open(AUDIO_FILE, 'rb') as file_obj:
+            results = self._make_sync_request(content=file_obj.read(),
+                                              max_alternatives=2,
+                                              use_gax=False)
+            second_alternative = results[1]
```

Comment:
Definitely let the file context manager close before doing the entire test. Like in #2648, just read the file in the context manager and then call `_make_sync_request()` after the context manager.